test/config/test_config.py

```diff
@@ -21,9 +15,17 @@ def loaded_config(tmp_path):
         "columns": { "vehicle_number": { "max": "NA", "min": 0 }, "maximum_speed" : {"max" : 150, "min" : 0}, "service_date" : {"max" : "NA", "min" : "1990-01-01"} }
```

Comment:
Can you add an extra columns field with no min or no max (meaning that the min or max field doesn't exist, not just "N/A"), and another with no min and no max, and add some check_bound assert cases for those? 

I suspect you would get a keyerror here. 

I think we should accept cases where the min and/or max field are left out entirely.